src/main/play-26/twirl/uk/gov/hmrc/hmrcfrontend/views/components/hmrcHeader.scala.html

```diff
@@ -94,20 +94,20 @@
       }
 
       @if(navigation.isDefined) {
-      <button type="button" role="button" class="govuk-header__menu-button js-header-toggle"
+      <button type="button" class="govuk-header__menu-button govuk-js-header-toggle"
           aria-controls="navigation"
           aria-label="Show or hide Top Level Navigation">Menu
       </button>
       <nav>
         <ul id="navigation"
           class="@toClasses("govuk-header__navigation ", navigationClasses)"
           aria-label="Top Level Navigation">
-          @for(item <- navigation.getOrElse(Seq.empty).filter(item => item.href.nonEmpty && item.text.nonEmpty)) {
```

Comment:
I think this is the bit that is making the integration test fail. Previously we were ignoring any entries that had an empty href or text. We're now including navigation items where href is empty.

Reply:
The filter probably needs changing to something like item => item.href.nonEmpty && (item.text.nonEmpty || item.content.nonEmpty)

Reply:
Worth adding a specific unit test or additional-fixture for this case.